examples/aws-production/cluster.lokocfg

```diff
@@ -13,6 +13,10 @@ variable "cluster_name" {
   default = "lokomotive-cluster"
 }
 
+variable "cluster_region" {
```

Comment:
this variable is _only_ used for S3 bucket location?

Reply:
This variable configures the region where the cluster is deployed.

The `AWS_DEFAULT_REGION` env variable is *only* used for the S3 bucket location in current Lokomotive.